internal/ignition/ignition.go

```diff
@@ -191,6 +192,28 @@ const highlyAvailableInfrastructureTopologyPatch = `---
 
 const tempNMConnectionsDir = "/etc/assisted/network"
 
+const bootReporterPath = "/assisted-boot-reporter.sh"
```

Comment:
We can use https://pkg.go.dev/embed instead of including it in the service image

Reply:
What makes it preferable here?

Reply:
Consistency. We already use it all over the service (most notably all the files included by `internal/ignition/templates.go` including files in ignition specifically too)

Also it avoids a change to the Dockerfile. We have copies of this Dockerfile everywhere maintained by the Agent team, our team, ACM pipelines, MCE, downstream SaaS, etc etc. So changes to the Dockerfile are super expensive from an engineering perspective because we have to make sure we do them everywhere, and do them right. We often get bug reports and complaints when those Dockerfiles don't get updated correctly.

Reply:
Done